src/System.Collections.Immutable/tests/ImmutableDictionaryTestBase.cs

```diff
@@ -347,93 +243,6 @@ public void TryGetKey()
             Assert.Equal(default(V), value);
         }
 
-        private IImmutableDictionary<TKey, TValue> AddTestHelper<TKey, TValue>(IImmutableDictionary<TKey, TValue> map, TKey key, TValue value) where TKey : IComparable<TKey>
-        {
-            Assert.NotNull(map);
-            Assert.NotNull(key);
-
-            IImmutableDictionary<TKey, TValue> addedMap = map.Add(key, value);
-            Assert.NotSame(map, addedMap);
-            ////Assert.Equal(map.Count + 1, addedMap.Count);
-            Assert.False(map.ContainsKey(key));
-            Assert.True(addedMap.ContainsKey(key));
-            AssertAreSame(value, addedMap.GetValueOrDefault(key));
-
-            this.VerifyAvlTreeState(addedMap);
-
-            return addedMap;
-        }
-
-        protected void AddAscendingTestHelper(IImmutableDictionary<int, GenericParameterHelper> map)
```

Comment:
Why would we need to move this to a nonnetstandard configuration without changing ImmutablesTestBase?

Reply:
ImmutableTestBase has about 16 derived types, some of which required no changes to its properties and we do in fact use them in some tests. For some other derived types, the implementation of those properties uses types not available in netstandard, which is why we only split those derived types.